README.md

```diff
@@ -1,3 +1,19 @@
+---
+### Repository sponsored by [Anvil](www.useanvil.com/developers) 
+
+![Horizontal Lockupblack](https://user-images.githubusercontent.com/293079/169453889-ae211c6c-7634-4ccd-8ca9-8970c2621b6f.png#gh-light-mode-only)
+![Horizontal Lockup copywhite](https://user-images.githubusercontent.com/293079/169453892-895f637b-4633-4a14-b997-960c9e17579b.png#gh-dark-mode-only)
+
+Anvil provides easy APIs for all things paperwork.
+1. [PDF filling API](https://www.useanvil.com/products/pdf-filling-api/) - fill out a PDF template with a web request and structured JSON data
```

Comment:
Periods at the end of each bullet? The last one has a period

Reply:
👍